src/cli/shared/__tests__/getConfig.spec.js

```diff
@@ -156,6 +156,25 @@ describe('Joi validate', () => {
     spy.mockRestore();
   });
 
+  it('should throw error if validate failed', () => {
+    const { MOCK_FILE_WITH_PLATFORM } = setup();
+    MOCK_FILE_WITH_PLATFORM.slack.accessToken = 12345;
+
+    importFresh.mockReturnValue(MOCK_FILE_WITH_PLATFORM);
+
+    const configPath = 'bottender.config.js';
+    const platform = 'messenger';
+
+    const spy = jest.spyOn(Joi, 'validate');
+
+    expect(() => {
+      getConfig(configPath, platform);
+    }).toThrow();
+
```

Comment:
Add an assertion`expect(spy).toHaveBeenCalled();`? 
Or the spy function is useless.